Classes/ItemsTab.lua

```diff
@@ -201,7 +201,7 @@ local ItemsTabClass = newClass("ItemsTab", "UndoHandler", "ControlHost", "Contro
 	-- Unique database
 	self.controls.uniqueDB = new("ItemDBControl", {"TOPLEFT",self.controls.itemList,"BOTTOMLEFT"}, 0, 76, 360, function(c) return m_min(244, self.maxY - select(2, c:GetPos())) end, self, main.uniqueDB[build.targetVersion], "UNIQUE")
 	self.controls.uniqueDB.y = function()
-		return self.controls.selectDBLabel:IsShown() and 98 or 76
+		return self.controls.selectDBLabel:IsShown() and 118 or 76
```

Comment:
The 76 at the end of the line needs to be changed to 96 otherwise the box collides with the item section above

Reply:
Also need to change line 213 to `return self.controls.selectDBLabel:IsShown() and 78 or 396` so it fits properly

Reply:
My bad never ran into the situation where "self.controls.selectDBLabel:IsShown()" was false, though for the second one I will change it but it shouldnt show up in rares so not sure why it needs to be changed